packages/strapi-design-system/src/DatePicker/DatePicker.tsx

```diff
@@ -102,6 +100,7 @@ export const DatePicker = ({
         aria-label={ariaLabel}
         disabled={disabled}
         type="text"
+        placeholder="--/--/--"
```

Comment:
I think I'd prefer if we not hardcode this placeholder.

Reply:
I'm going to remove this for now, it's a prop in another PR.